bower.json

```diff
@@ -1,9 +1,10 @@
 {
   "name": "ember-couchdb-kit",
   "description": "An Ember.js adapter for Apache CouchDB",
+  "version": "1.0.2",
   "repository": {
     "type": "git",
-    "url": "git://github.com/zatvobor/ember-couchdb-kit.git"
+    "url": "git://github.com/ValidUSA/ember-couchdb-kit.git"
```

Comment:
We decided that our needs for this project differ from what everyone may need and therefore didn't plan on sending changes back. We have specific needs that aren't in most use cases.